lib/sycamore/sycamore/docset.py

```diff
@@ -332,6 +333,24 @@ def spread_properties(self, props: list[str], **resource_args) -> "DocSet":
         plan = SpreadProperties(self.plan, props, **resource_args)
         return DocSet(self.context, plan)
 
+    def assign_doc_properties(self, element_type:str, property_name:str, **resource_args) -> "DocSet":
```

Comment:
It seems that the source element is the first of the specified element_type. You should add that to the docstring.

Reply:
thanks for catching this, Fixing it in the next push.